### PR TITLE
Nerf budget insulated gloves

### PR DESCRIPTION
**Pull request by liltenhead**
## About the PR
Overall lowers the chances of budget gloves being insulated, and increases the odds of them doing more damage to the user. 

## Why / Balance
Budget insulated gloves should rarely be insulated, and shouldn't be risk free to test them. Basically every shift you'll see someone hacking a door and taking 22 damage at worst, so they can test every glove in the Youtool before having to go to medical. Increasing the damage multipliers, and lowering the odds in general should make this a bit less desirable to do. 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: reduced the odds of budget insulated gloves being insulated and increased the damage multipliers on bad gloves.

### CODE REVIEW

**Comment by UbaserB** [comment]
hahaa based

**Comment by EmoGarbage404** [comment]
With current power damage values, what's the chance that using budget insuls would just kill the player? LV damage can be quite high and testing insuls shouldn't just instantly kill someone.

**Comment by liltenhead** [comment]
People just either hack doors or vending machines to test them. You would take 60 damage max with the worst rolled gloves on a vending machine.

**Comment by deltanedas** [comment]
budget rollers in shambles

**Comment by liltenhead** [comment]
> failing to hack doors due to budget insuls should blow your hands off :nodders:

I honestly hope that happens when surgery gets added

**Comment by PJB3005** [comment]
I am the #1 advocate of hacking and I approve of this PR.

Partially because it means medbay players get more people to treat at round start.

**Comment by Holinka4ever** [comment]
First of all, an offtopic thought: a  lot of people abusing budget insulas, it seems in my opinion truely wrong, but its not a problem with chances when insulas might be good or bad, problem is...

At second on topic: ...starting quantity of gloves itself. Station itself has ginourmous count of budget gloves from right to left. Of corse people would abuse it due being truely infinite tries.

So my solution is to decrease budget insulas in you-tool, or even put them into contraband segment of vendor.

File: Resources/Prototypes/Entities/Clothing/Hands/colored.yml [hunk 1]
```diff
@@ -178,19 +178,19 @@
...
-      - 1.5
-      - 1.5
       - 1.5
       - 1.5
+      - 2
+      - 2.5
+      - 2.5
+      - 3
+      - 3.5
+      - 4
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Hands/colored.yml:194]: This is the worst random code I have ever seen.

**Pull request merged**